### DISCUSSION TITLE
How to style a Filament 'plugin'

### DISCUSSION THREAD

**Discussion by rabol** [filamentphp/filament#543] [Help]
Hi

Tailwind is not one of my strong points so I need some help please :)

I have created a log viewer for Filament and now I'm down to the last 'bit', where I need to show the details of a log entry.

This is what I have so far:

![Screen Shot 2021-08-24 at 17 09 26](https://user-images.githubusercontent.com/1177191/130642196-0ab35844-b67b-49dd-b694-bba63dfbd7dc.png)

![Screen Shot 2021-08-24 at 17 09 37](https://user-images.githubusercontent.com/1177191/130642605-03b0ac25-9521-472a-8bd6-ed4570115f07.png)

Now, do I show the 'details' as a pop up or do I use collapsable div's or ??

My biggest issue is that I have tried to copy paste a lot of Tailwind components into the views, but it simply does not work as most of the css classes does not exists.

So how do I go about this ?

**Comment by danharrin** [comment]
You need to watch a video / read an article on how to compile a Tailwind .css file. I suggest using Laravel Mix. Once you've done this, you can register the new styles using your plugin service provider.

**Reply by rabol** [reply]
I do know how to make/build Tailwind, however I still need to figure out how to include Filament specific changes. 
with a standard build of tailwind and registration of the assets I can make it work, but it override Filament changes

**Reply by goeil** [reply]
I think a document would be appreciated to be able to use FULL tailwind in the admin project.

I cannot manage for now to use tailwind classes (I know that only some of them are available, but how can we use all the others ?)

I'm loosing my mind between postcss, vite, AppServiceProvider, tailwind.config.js, app.css…

Any tip will be appreciated!
Thanks

**Reply by danharrin** [reply]
Hey, if you just want access to all Tailwind classes and you don't have a plugin that you want to distribute, then you should register a theme - https://filamentphp.com/docs/2.x/admin/appearance#building-themes

This will compile all custom classes within /resources/views
